swagger.yaml

```diff
@@ -12,7 +12,7 @@ schemes:
 tags:
   - name: "public"
 paths:
-  /legacy:
+  /articles/legacy:
```

Comment:
Quick question: the path should not include the 'v1'? i.e. /v1/articles/legacy

Reply:
Yes, it's a bit further up, in line 9 (unchanged)

[basePath: "/v1"](https://github.com/ONSdigital/dp-articles-api/blob/cb913da034b8f43ad7a9354be5cea0e6f486c04c/swagger.yaml#L9)